src/main/java/frc/robot/commands/OneBar/PID.java

```diff
@@ -13,20 +13,19 @@ public class PID extends CommandBase {
   private final OnebarSubsystem onebarsubsystem;
   private final PIDController PIDo = new PIDController(Constants.kP, Constants.kI, Constants.kD);
 
-  double setPoint;
+  public double setPoint;
 
   /** Creates a new PID. */
-  public PID(OnebarSubsystem subsystem) {
+  public PID(OnebarSubsystem subsystem, double pointSet) {
```

Comment:
If you want to use the same name in the parameter as the field in the class, you can use the `this` keyword.

```java
public PID(OnebarSubsystem onebarsubsystem, double setPoint) {
  this.onebarsubsystem = onebarsubsystem;  
  this.setPoint = setPoint;
  ...
```